coder_sniffer/DrupalPractice/Sniffs/General/OptionsTSniff.php

```diff
@@ -81,6 +81,21 @@ public function process(File $phpcsFile, $stackPtr)
             $statementEnd = $tokens[$arrayToken]['bracket_closer'];
         }
 
+        // We want to find if the element "#options" belongs to a form element.
+        // Array with selectable options for a form element.
+        $formElements = [
+          "'checkboxes'",
+          "'radios'",
+          "'select'",
+          "'tableselect'",
+        ];
+        // Find beggining of the array containing "#options" element.
+        $startArray = $phpcsFile->findPrevious(T_EQUAL, $stackPtr, null, false, "=");
```

Comment:
Single quotes can be used here for the '=' value.

We should add an $end value here to scope the array definition, as this search will currently keep searching until it finds a '=', which could result in finding a different form element. For example, the following code.
```
$form['lorem'] = [
  '#type' => 'checkboxes',
  '#options' => [
    'lorem' => 'Lorem',
  ],
];

$form['ipsum']['#options'] = [
  'attributes' => [
    'target' => '_blank',
  ],
];
```
Will result in:
```
--------------------------------------------------------------------------------
  4 | WARNING | #options values usually have to run through t() for translation
 10 | WARNING | #options values usually have to run through t() for translation
--------------------------------------------------------------------------------
```

Reply:
I assume we can look for a previous variable here?